test/test-page-service.ts

```diff
@@ -0,0 +1,102 @@
+import { Yok } from "mobile-cli-lib/yok";
+import { PageService } from "../lib/services/page-service";
+import { GitService } from "../lib/services/git-service";
```

Comment:
file: 'file:///d%3A/GitHub/nativescript-starter-kits/test/test-page-service.ts'
severity: 'Error'
message: 'Import sources within a group must be alphabetized. (ordered-imports)'
at: '3,1'
source: 'tslint'